RecoTauTag/TauTagTools/test/training/skimming/buildSignalDataSet_cfg.py

```diff
@@ -199,7 +199,7 @@
 )
 
 # Produce and save PiZeros for the signal jets
-import RecoTauTag.RecoTau.RecoTauPiZeroProducer_cfi as PiZeroProd
+import RecoTauTag.RecoTau.RecoTauPiZeroProducer_cff as PiZeroProd
 process.signalJetsRecoTauPiZeros = PiZeroProd.ak5PFJetsRecoTauPiZeros.clone(
```

Comment:
Same here:  ak5PFJetsRecoTauPiZeros is non existing any more in that file, since February 2014. I imagine that these tests are outdated: maybe at some point one could decide to also either update or remove them.